src/classes/CCON_ConnectionBackfill_TEST.cls

```diff
@@ -30,7 +30,7 @@
 /**
 * @author Salesforce.org
 * @date 2016
-* @group CourseEnrollments
+* @group Course Enrollments
```

Comment:
I updated several files when reviewing our ApexDocs and noticing some inconsistencies. If we don't want to include these changes in this PR, I can just revert the one commit.

Reply:
No this is fine.